converters/toZigbee.js

```diff
@@ -147,6 +147,35 @@ const converters = {
             await entity.read('genOnOff', ['onOff']);
         },
     },
+    TYZB01_on_off: {
+        key: ['state'],
+        convertSet: async (entity, key, value, meta) => {
+            if (typeof value !== 'object') {
+                return converters.on_off.convertSet(entity, key, value, meta);
+            }
+            utils.validateValue(value.state, ['on', 'off']);
+            const timeInSeconds = Number(value.time_in_seconds);
```

Comment:
Instead of `time_in_seconds` can you rename this to `transition` (to be more in line with other devices).